docs/storing-querying/loading-process/index.md

```diff
@@ -25,11 +26,14 @@ We load data into Redshift using the [RDB Loader](/docs/pipeline-components-and-
 
   </TabItem>
   <TabItem value="bigquery" label="BigQuery">
-
-We load data into BigQuery using the [BigQuery Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/index.md).
```

Comment:
Could you bring back this line? It’s useful as a link to the BQ Loader page, and also to explain what Versions 2.x/1.x refer to.